flume-ng-sinks/flume-ng-hbase-sink/src/main/java/org/apache/flume/sink/hbase/AsyncHBaseSink.java

```diff
@@ -329,6 +330,11 @@ public Status process() throws EventDeliveryException {
 
   @Override
   public void configure(Context context) {
+    if (!HBaseVersionCheck.hasVersionLessThan2(logger)) {
+      throw new ConfigurationException(
+              "HBase major version number must be less than 2 for asynchbase sink. " +
+                      "Use hbase2-sink instead.");
```

Comment:
Is it going to be hbase2-sink for both asynchbase and hbase-sink in the next version? Ie. are they going to be merged?

Reply:
hbase2-sink is going to be a new sink based on hbase-sink. After adding hbase2-sink the flume will be use asynchbase-sink and hbase-sink with hbase-client1.x and will be able to use hbase2-sink with hbase-client2.x versions.
Unfortunately asynchbase library is not compatible with hbase 2.0